### DISCUSSION TITLE
How to do sub queries?

### DISCUSSION THREAD

**Discussion by felinto-dev** [prisma/prisma#12026] [Q&A] [answered]
Could someone give me an example of how to use subqueries in Prisma?

https://www.tutorialspoint.com/sql/sql-sub-queries.htm

**Comment by nurul3101** [comment, marked as answer]
Hey @felinto-dev 👋🏻 ,
Unfortunately, Prisma doesn't support subqueries natively as of now, however you could use [`raw queries`](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access#raw-queries-with-relational-databases) to use subqueries.
Here's a [Feature Request](https://github.com/prisma/prisma/issues/5617) for supporting subqueries. 
Can you please leave a 👍🏻 to the issue? 
It helps our product and engineering team in prioritising it.

**Reply by felinto-dev** [reply]
Of course!
Thank you!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
